lib/ansible/modules/apt.py

```diff
@@ -63,44 +63,43 @@
     default: 'no'
   default_release:
     description:
-      - Corresponds to the C(-t) option for I(apt) and sets pin priorities
+      - Corresponds to the C(-t) option for C(apt) and sets pin priorities.
     aliases: [ default-release ]
     type: str
   install_recommends:
     description:
-      - Corresponds to the C(--no-install-recommends) option for I(apt). V(true) installs recommended packages.  V(false) does not install
+      - Corresponds to the C(--no-install-recommends) option for C(apt). V(true) installs recommended packages. V(false) does not install
         recommended packages. By default, Ansible will use the same defaults as the operating system. Suggested packages are never installed.
     aliases: [ install-recommends ]
     type: bool
   force:
     description:
-      - 'Corresponds to the C(--force-yes) to I(apt-get) and implies O(allow_unauthenticated=yes) and O(allow_downgrade=yes)'
-      - "This option will disable checking both the packages' signatures and the certificates of the
-        web servers they are downloaded from."
-      - 'This option *is not* the equivalent of passing the C(-f) flag to I(apt-get) on the command line'
+      - 'Corresponds to the C(--force-yes) to I(apt-get) and implies O(allow_unauthenticated=yes) and O(allow_downgrade=yes).'
+      - "This option will disable checking both the packages' signatures and the certificates of the web servers they are downloaded from."
+      - 'This option *is not* the equivalent of passing the C(-f) flag to C(apt-get) on the command line.'
```

Comment:
i don't see why this is flagged as changed

Reply:
Punctuation.